services/hooks/src/listeners.js

```diff
@@ -84,7 +84,12 @@ class HookListeners {
         hookId: hookId,
       }, true);
       if (latestHook) {
-        await this.taskcreator.fire(latestHook, {firedBy: 'pulseMessage', payload});
+        try {
+          await this.taskcreator.fire(latestHook, {firedBy: 'pulseMessage', payload});
```

Comment:
Nit: Would it make more sense to have the try/catch inside taskcreator if it should never throw an error anyway?

Reply:
For both `Hooks.triggerHook` and the time-based scheduling, we *do* want to handle this error (the former to pass it along to the user; the latter to decide whether to retry the fire on the next iteration).  I thought of adding a `throwError: ..` option to `taskcreator.fire`, but this seemed a bit more natural.